icu4c/source/i18n/unicode/rbnf.h

```diff
@@ -297,7 +297,7 @@ enum URBNFRuleSetTag {
  *   </tr>
  *   <tr>
  *     <td>x.0:</td>
- *     <td>The rule is a <em>master rule</em>. If the full stop in
+ *     <td>The rule is a <em>primary rule</em>. If the full stop in
```

Comment:
I would like @grhoten to chime in on renaming "master rule" in RBNF. Maybe even @richgillam has an opinion.

Reply:
Can we replace "master" with "default" rule in this entire header? It's not really master nor primary. We don't really have tertiary types of rules, which is why I don't want to say primary nor secondary.

Reply:
Another reason to not say primary is that the other rules are the primary ones and not this one.

Reply:
Wow, this brings back memories.  Are we talking about the `.:` rule that gets used to split the number into integral and fractional parts?  If so, I'm fine with changing it from "master", and I agree with George that "primary" doesn't feel right either.  Maybe because this is a very specialized rule that's just used to split the number into integral and fractional parts, we should give it a specific name, such as "decimal point rule."

Of course, if I've misunderstood what we're talking about here, feel free to ignore me.  :-)

Reply:
@richgillam actually your information is a little out of date. This is in reference to selecting alternate decimal points. So you can say “point” or “comma” without duplicating the rules into a bunch of locales.

The full stop is for all locales, unless an alternate one matches, like a comma. This is important for Spanish. I added this bit of functionality.

Reply:
Thanks for bringing me up to date.  Based on what you're saying, "master rule" and "primary rule" both sound even worse and "decimal point rule" sounds better.  But I'll go with whatever.

Reply:
Well these are decimal point rules. Just one of them is the default (this one with the full stop) in the absence of a localized one.

Reply:
Okay, that makes sense.  Thanks for the clarifications.

Reply:
So you recommend `The rule is a <em>default rule</em>.` ?

Reply:
Yes, "default rule" is a good name for now.

Reply:
Done. Thx for your suggestion.